ArduCopter/mode_acro.cpp

```diff
@@ -120,82 +120,82 @@ void ModeAcro::get_pilot_desired_rates_cds(float roll_in_norm, float pitch_in_no
     // calculate roll, pitch rate requests
 
     // roll rate request with input expo applied
-    rate_bf_request_cds.x = g2.command_model_acro_rp.get_rate() * 100.0 * input_expo(roll_in_norm, g2.command_model_acro_rp.get_expo());
+    rate_bf_request_rads.x = radians(g2.command_model_acro_rp.get_rate()) * input_expo(roll_in_norm, g2.command_model_acro_rp.get_expo());
 
     // pitch rate request with input expo applied
-    rate_bf_request_cds.y = g2.command_model_acro_rp.get_rate() * 100.0 * input_expo(pitch_in_norm, g2.command_model_acro_rp.get_expo());
+    rate_bf_request_rads.y = radians(g2.command_model_acro_rp.get_rate()) * input_expo(pitch_in_norm, g2.command_model_acro_rp.get_expo());
 
     // yaw rate request with input expo applied
-    rate_bf_request_cds.z = g2.command_model_acro_y.get_rate() * 100.0 * input_expo(yaw_in_norm, g2.command_model_acro_y.get_expo());
+    rate_bf_request_rads.z = radians(g2.command_model_acro_y.get_rate()) * input_expo(yaw_in_norm, g2.command_model_acro_y.get_expo());
 
     // calculate earth frame rate corrections to pull the copter back to level while in ACRO mode
 
     if (g.acro_trainer != (uint8_t)Trainer::OFF) {
 
         // get attitude targets
-        const Vector3f att_target_euler_cd = attitude_control->get_att_target_euler_cd();
+        const Vector3f att_target_euler_rad = attitude_control->get_att_target_euler_rad();
 
         // Calculate trainer mode earth frame rate command for roll
-        float roll_angle_cd = wrap_180_cd(att_target_euler_cd.x);
-        rate_ef_level_cds.x = -constrain_float(roll_angle_cd, -ACRO_LEVEL_MAX_ANGLE, ACRO_LEVEL_MAX_ANGLE) * g.acro_balance_roll;
+        float roll_angle_rad = wrap_PI(att_target_euler_rad.x);
+        rate_ef_level_rads.x = -constrain_float(roll_angle_rad, cd_to_rad(-ACRO_LEVEL_MAX_ANGLE), cd_to_rad(ACRO_LEVEL_MAX_ANGLE)) * g.acro_balance_roll;
```

Comment:
similar to Flip mode we could consider changing the ACRO_LEVEL_MAX_ANGLE definition to be in radians.  The downside of doing this though is it would affect any external people/companies that have these definitions in their hwdef files... I've never heard of anyone overriding these values in an hwdef though

Reply:
I would like to do this when we convert Sport mode.